File: decision-matrix/frontend/src/lib/taskLog/jobLabels.ts
export const JOB_TYPE_LABELS: Record<string, string> = {
  sand_logistics_analyze: 'Логистика песка',
  poi_analyze_all: 'Анализ окружения',
  autoroad_connect: 'Автодороги / сеть',
  import_file: 'Импорт файла',
  pad_earthwork_compute: 'Земляные работы куста',
  pad_placement_compute: 'Оптимизация кустов',
  pad_placement_apply: 'Применение кустов',
  well_trajectory_compute: 'Anti-collision (SF)',
  well_trajectory_import: 'Импорт инклинометрии',
};

/** Подписи для автоматически созданных HTTP-flow (последние сегменты path). */
export const HTTP_FLOW_PATH_LABELS: Record<string, string> = {
  'pois/analyze-all': 'Анализ окружения',
  'sand-logistics/analyze': 'Логистика песка',
  'infrastructure/autoroad-connect': 'Соединение автодорогами',
  'autoroad-network/request': 'Сеть автодорог: подготовка',
  'autoroad-network/compute': 'Сеть автодорог: расчёт',
  'autoroad-network/apply': 'Сеть автодорог: применение',
  'pad-placement/compute': 'Оптимизация кустов: расчёт',
  'pad-placement/apply': 'Оптимизация кустов: применение',
};

export const JOB_STATUS_LABELS: Record<string, string> = {
  pending: 'В очереди',
  running: 'Выполняется',
  completed: 'Завершена',
  failed: 'Ошибка',
  cancelled: 'Отменена',
};

export const ACTIVE_JOB_STATUSES = new Set(['pending', 'running']);

export function jobTypeLabel(jobType: string): string {
  return JOB_TYPE_LABELS[jobType] ?? jobType;
}

export function jobStatusLabel(status: string): string {
  return JOB_STATUS_LABELS[status] ?? status;
}

File: decision-matrix/backend/app/assistant/chat/job_labels.py
"""Russian labels for project background jobs (mirror frontend jobLabels.ts)."""

from __future__ import annotations

JOB_TYPE_LABELS: dict[str, str] = {
    "sand_logistics_analyze": "Логистика песка",
    "poi_analyze_all": "Анализ окружения",
    "autoroad_connect": "Автодороги / сеть",
    "import_file": "Импорт файла",
    "pad_earthwork_compute": "Земляные работы куста",
    "well_trajectory_compute": "Anti-collision (SF)",
    "well_trajectory_import": "Импорт инклинометрии",
    "pad_placement_compute": "Оптимизация кустов",
    "pad_placement_apply": "Применение кустов",
    "line_elevation_profile_compute": "Профиль высот линий",
}

JOB_STATUS_LABELS: dict[str, str] = {
    "pending": "В очереди",
    "running": "Выполняется",
    "completed": "Завершена",
    "failed": "Ошибка",
    "cancelled": "Отменена",
}


def job_type_label_ru(job_type: str) -> str:
    return JOB_TYPE_LABELS.get(job_type, job_type)


def job_status_label_ru(status: str) -> str:
    return JOB_STATUS_LABELS.get(status, status)
